src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/ArgumentAnalysisResultKind.cs

```diff
@@ -9,6 +9,7 @@ internal enum ArgumentAnalysisResultKind : byte
         NoCorrespondingParameter,
         FirstInvalid = NoCorrespondingParameter,
         NoCorrespondingNamedParameter,
+        DuplicateNamedArguments,
```

Comment:
>DuplicateNamedArguments [](start = 8, length = 23)

Should we use singular as the other members, i.e. "DuplicateNamedArgument"?

Reply:
I chose the plural form as you cannot have one duplicate argument :)

---
In reply to: [178667114](https://github.com/dotnet/roslyn/pull/25730#discussion_r178667114) [](ancestors = 178667114)

Reply:
> I chose the plural form as you cannot have one duplicate argument 

Of course I can, the second one is a duplicate, the first one is not.

---
In reply to: [178707798](https://github.com/dotnet/roslyn/pull/25730#discussion_r178707798) [](ancestors = 178707798,178667114)

Reply:
Alright. I'll fix.

---
In reply to: [178710288](https://github.com/dotnet/roslyn/pull/25730#discussion_r178710288) [](ancestors = 178710288,178707798,178667114)